index.html

```diff
@@ -2,12 +2,189 @@
+          </li>
+          <li>
+            <h2 class = "EducationHeader">Rockey Academy: Software Engineering Bootcamp</h2>
+            <span>From Sept 2022 to Apr 2023</span>
+            <p class = "educationDescription">
+              Advanced understanding of HTML, CSS, Javascript, Typescript and SQL<br>
+              Experienced in Javascript Frameworks: React, nodeJS, Express.js
+              Experienced in Firebase, Sequalize, Auth0
+              Completed 4 projects:
+              <ul class = "projectList">
+                <li>Project 1 (Frontend): Build a Frontend App</li>
+                <li>Project 2 (Full-stack): Build a Full-stack App (Firebase)</li>
+                <li>Project 3 (Backend): Build a Full-stack app (Express)</li>
+                <li>Project 4: Capstone Project</li>
+              </ul>
+            </p>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div class = "centerDiv">
+      <div id = "Projects">
+        <h1 id = "bigHeader">Projects</h1>
+        <div class = "rows">
+          <div class = "projectShowcase">
+            <h1 style="text-align: center;">RA Basics Project 1: <br>Rock, Paper, Scissors</h1>
+            <img class = "shrink" src = "./images/RPS.png">
+            <p>
+              Simple Rock Paper Scissor Game with AI<br>
+              <span>Features:</span><br>
+              <ul style = "font-size: 20px;">
+                <li>Normal Mode</li>
+                <li>Reverse Mode</li>
+                <li>Muk-jji-ppa</li>
+                <li>AI vs AI</li>
+                <li>Recorded Stats</li>
+                <li>Mode control: Choose your mode</li>
+              </ul>
+              <a href="">Link to Github Repo</a>
+            </p>
+          </div>
+          <div class = "projectShowcase">
+            <h1 style="text-align: center;">RA Basics Project 2: <br>Beat That!</h1>
+            <img class = "shrink" src = "./images/diceroll.jfif">
+            <p>
+              Multiplayer Random Dice Game.<br>
+              Player with the highest-combined number wins.<br>
+              <span>Features:</span><br>
+              <ul style = "font-size: 20px;">
+                <li>Scoring</li>
+                <li>Leaderboard</li>
+                <li>Lowest-Combined Mode</li>
+                <li>Highest-Combined Mode</li>
+                <li>Auto-generated combination</li>
+                <li>Varied number of players and dice</li>
+              </ul>
+              <a href="https://github.com/LoyChaiEe/coding-basics/tree/main/Project2/beat-that">Link to Github Repo</a>
+            </p>
+          </div>
+          <div class = "projectShowcase">
+            <h1 style="text-align: center;">RA Basics Project 3: <br>BlackJack</h1>
+            <img class = "shrink" src = "./images/blackjack.png">
+            <p>
+              Multiplayer BlackJack <br>
+              <span>Features:</span><br>
+              <ul style = "font-size: 20px;">
```

Comment:
let's use our CSS files for styles and avoid in-line styles like so across our whole HTML!